src/coreclr/vm/reflectioninvocation.cpp

```diff
@@ -492,6 +492,11 @@ extern "C" void QCALLTYPE RuntimeMethodHandle_InvokeMethod(
     // If an exception occurs a gc may happen but we are going to dump the stack anyway and we do
     // not need to protect anything.
 
+    // should we use malloc if the size is too big?
```

Comment:
I do not think that's necessary. We have no guarantees for how many copies of arguments or return buffers get allocated on the stack, as long as it is a small constant number.

Reply:
Thanks! removed the comment